### PR TITLE
Add posix-spawn package

### PR DESCRIPTION
**Pull request by xprazak2**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.20
* [ ] 1.19
* [ ] 1.18

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by xprazak2** [comment]
Needed for https://github.com/theforeman/foreman_scap_client/pull/25

**Comment by xprazak2** [comment]
@ekohl, any thoughts on how to get the repoclosure to pass?

**Comment by ekohl** [comment]
[test rpm]

**Comment by xprazak2** [comment]
I removed epoch and autogenerated comments.

**Comment by xprazak2** [comment]
All green now.

File: packages/client/rubygem-posix-spawn/rubygem-posix-spawn.spec [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+mkdir -p %{buildroot}%{gem_extdir_mri}/lib
+
+mv %{buildroot}%{gem_libdir}/posix_spawn_ext.so \
+    %{buildroot}%{gem_extdir_mri}/lib/
+
+#cp -a ./%{gem_extdir_mri}/* %{buildroot}%{gem_extdir_mri}/
```
Comment (ekohl) [inline packages/client/rubygem-posix-spawn/rubygem-posix-spawn.spec]: Could remove the commented code?

File: packages/client/rubygem-posix-spawn/rubygem-posix-spawn.spec [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Generated from posix-spawn-0.3.13.gem by gem2rpm -*- rpm-spec -*-
+# template: nonscl
+%global gem_name posix-spawn
+
+#%define rubyabi 1.8
```
Comment (ekohl) [inline packages/client/rubygem-posix-spawn/rubygem-posix-spawn.spec]: Could you remove the commented code?

File: packages/client/rubygem-posix-spawn/rubygem-posix-spawn.spec [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+rm -rf %{buildroot}%{gem_instdir}/ext/
+
+mkdir -p %{buildroot}%{_bindir}
+cp -pa .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+
+%files
+%dir %{gem_instdir}
+%{_bindir}/posix-spawn-benchmark
```
Comment (ekohl) [inline packages/client/rubygem-posix-spawn/rubygem-posix-spawn.spec]: I wonder if this is really needed

File: packages/client/rubygem-posix-spawn/rubygem-posix-spawn.spec [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+%exclude %{gem_instdir}/.gitignore
+%{gem_instdir}/bin
+%{gem_libdir}
+%exclude %{gem_cache}
+%{gem_spec}
+#%{gem_extdir_mri}
```
Comment (ekohl) [inline packages/client/rubygem-posix-spawn/rubygem-posix-spawn.spec]: Commented code

**Comment by xprazak2** [comment]
This will not be needed after all, closing.

**Pull request closed without merging**